app/src/main/res/layout/activity_onboarding.xml

```diff
@@ -18,134 +18,142 @@
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <Button
-            android:id="@+id/proceedToSetup"
-            style="@style/BlueFullWidthButton"
-            android:layout_alignParentBottom="true"
-            android:layout_centerHorizontal="true"
-            android:layout_marginBottom="@dimen/dp_40"
-            android:fontFamily="@font/roboto_regular"
-            android:text="@string/actions_get_started"
-            android:visibility="gone" />
-
-        <TextView
-            android:id="@+id/skipActionView"
-            style="@style/WhiteLinkAction"
-            android:layout_alignParentBottom="true"
-            android:layout_centerHorizontal="true"
-            android:layout_marginBottom="@dimen/dp_40"
-            android:text="@string/actions_skip"
-            tools:background="@color/blue"
-            tools:textColor="@android:color/white" />
-
-        <androidx.viewpager.widget.ViewPager
-            android:id="@+id/onboardingPager"
+        <RelativeLayout
+            android:id="@+id/onboardingLayout"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_above="@+id/pageIndicatorView"
-            android:layout_alignParentTop="true"
-            android:layout_marginBottom="@dimen/dp_30" />
+            android:visibility="@{onboardingSetupViewModel.hideOnboardingAndShowPasscodeSetupView == true ? View.GONE : View.VISIBLE}">
 
-        <com.saltedge.authenticator.widget.pageindicator.DotsPageIndicatorView
-            android:id="@+id/pageIndicatorView"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_alignParentBottom="true"
-            android:layout_centerHorizontal="true"
-            android:layout_marginBottom="@dimen/dp_124"
-            app:selectedColor="@color/blue" />
-    </RelativeLayout>
-</FrameLayout>
+            <Button
+                android:id="@+id/proceedToSetup"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/dp_48"
+                android:layout_alignParentBottom="true"
+                android:layout_centerHorizontal="true"
+                android:layout_marginStart="@dimen/dp_32"
+                android:layout_marginEnd="@dimen/dp_32"
+                android:layout_marginBottom="@dimen/dp_40"
+                android:backgroundTint="@color/button_color_selector"
+                android:elevation="@dimen/dp_30"
+                android:fontFamily="@font/roboto_medium"
+                android:text="@string/actions_get_started"
+                android:textAllCaps="false"
+                android:textColor="@android:color/white"
+                android:textSize="@dimen/text_14"
+                android:visibility="@{onboardingSetupViewModel.hideSkipViewAndShowProceedView == true ? View.VISIBLE : View.GONE}" />
```

Comment:
how to test this condition?

Reply:
You can view the test for the following method `onOnboardingPageSelected`